daemon/src/db.rs

```diff
@@ -283,6 +283,56 @@ pub async fn load_all_cfd_ids(conn: &mut PoolConnection<Sqlite>) -> Result<Vec<O
     Ok(ids)
 }
 
+/// Loads cfd that are not final yet
+///
+/// Finality can happen by:
+/// 1. Recording the confirmation of a payout (spend) transaction on the blockchain
+///     Cases: Collaborative settlement, CET, Refund
+/// 2. The CFD failing early meaning it becomes irrelevant for processing
+///     Cases: Setup failed, Taker's take order rejected
+pub async fn load_non_final_cfd_ids(conn: &mut PoolConnection<Sqlite>) -> Result<Vec<OrderId>> {
```

Comment:
Any reason we use the wording "non_final" here instead of "open"?

Reply:
I just tried to be more precise because we use `Open` in a very ambiguous way. E,g. in the projection we can have multiple states after `Open`, until we finally reach a final state.

Reply:
I am happy to change it to `Open` if you think it help...?

Reply:
I think I'd prefer `Open`. Other opinions?

Reply:
IMHO "non_final" is a bit better due to issue outlined by @da-kami earlier - there are multiple states around "Open" (some *before* and some *after*, which makes it even more confusing), the current naming is a bit more unambiguous.
At the end of the day though, we're in a different context here, so one could plausibly ignore projection naming here (on the database level there are no "states").

Reply:
In the end I opted for renaming it to `open` to avoid introducing yet another terminology. 
I adapted the rustdoc of the function to be very specific what `open` means in this context.

I think our discussion shows that we should cleanup the projection API because it adds confusion :)

Reply:
> I think our discussion shows that we should cleanup the states in the projection API because it adds confusion :)

Yes please <3